Content-type: text/xml

<?xml version="1.0" encoding="utf-8"?>
<testing-report run-id="194" judge-id="4" status="SE" scoring="ACM" run-tests="1" contest-id="3103" real-time-available="yes" max-memory-used-available="yes" correct-available="yes" failed-test="1" time-limit-ms="2000" real-time-limit-ms="5000" marked-flag="no" >
  <uuid>b85aa183-9e01-4195-82f4-793d08ae4b7f</uuid>
  <host>localhost.localdomain</host>
  <cpu-model>Intel(R) Pentium(R) CPU 3550M @ 2.30GHz</cpu-model>
  <cpu-mhz>2294.738</cpu-mhz>
  <compiler_output></compiler_output>
  <tests>
    <test num="1" status="SE" time="285" real-time="326" max-memory-used="18702336" >
  <program-stats-str>{ utime=256, stime=29, ptime=285, rtime=326, maxvsz=18702336, maxrss=19353600, nvcsw=2, nivcsw=42 }</program-stats-str>
      <input size="12">3 2
1 2
1 3
</input>
      <output size="0"></output>
      <stderr size="381">Can't open '/usr/share/abrt/conf.d/plugins/python3.conf': Operation not permitted
Can't open '/etc/abrt/plugins/python3.conf': Operation not permitted
Traceback (most recent call last):
  File &quot;./000194&quot;, line 23, in &lt;module&gt;
    dfs(i, p)
  File &quot;./000194&quot;, line 16, in dfs
    dfs(u, v)
  File &quot;./000194&quot;, line 15, in dfs
    if used[u] == 0:
IndexError: list index out of range
</stderr>
    </test>
  </tests>
</testing-report>
